services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/ISwitchManager.java

```diff
@@ -197,13 +212,30 @@ long installTransitFlow(DatapathId dpid, String flowId, Long cookie, int inputPo
      * @param inputVlanId    vlan to match on inputPort
      * @param outputVlanId   set vlan on packet before forwarding via outputPort; 0 means not to set
      * @param outputVlanType type of action to apply to the outputVlanId if greater than 0
+     * @param enableLldp     if True LLDP packets will be send to LLDP rule
      * @return transaction id
      * @throws SwitchOperationException Switch not found
      */
     long installOneSwitchFlow(final DatapathId dpid, final String flowId, final Long cookie,
                                                       final int inputPort, final int outputPort, int inputVlanId,
                                                       int outputVlanId, final OutputVlanType outputVlanType,
-                                                      final long meterId) throws SwitchOperationException;
+                                                      final long meterId, boolean enableLldp)
+            throws SwitchOperationException;
+
+    /**
+     * Installs a flow to catch LLDP packets.
+     *
+     * @param dpid           datapathId of the switch
+     * @param flowId         flow id
+     * @param cookie         flow cookie
+     * @param inputPort      port to expect packet on
+     * @param outputVlanId   vlan to match on output vlan id
+     * @param meterId        flow meter ID
+     * @return transaction id
+     * @throws SwitchOperationException Switch not found
+     */
+    long installOneSwitchLldpFlow(DatapathId dpid, String flowId, long cookie, int inputPort, int outputVlanId,
```

Comment:
this should be the same as lldp for ingress

Reply:
fixed